modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastFragment.kt

```diff
@@ -611,6 +611,7 @@ class PodcastFragment : BaseFragment(), Toolbar.OnMenuItemClickListener {
                 theme.toggleDarkLightThemeActivity(activity as AppCompatActivity)
                 true
             }
+            it.menu.findItem(R.id.media_route_menu_item).isVisible = !FeatureFlag.isEnabled(Feature.UPNEXT_IN_TAB_BAR)
```

Comment:
I also removed the cast icon from the Podcast and Cloud screens as the icon is now visible on the mini-player.

Reply:
I reverted it in https://github.com/Automattic/pocket-casts-android/pull/2101/commits/829cc26f36d6b699367cced67dca540c79e58a26 as I can see the cast icon in iOS app for these screens.